crates/egui/src/containers/collapsing_header.rs

```diff
@@ -425,8 +425,8 @@ impl CollapsingHeader {
     /// Explicitly set the source of the [`Id`] of this widget, instead of using title label.
     /// This is useful if the title label is dynamic or not unique.
     #[inline]
-    pub fn id_source(mut self, id_source: impl Hash) -> Self {
-        self.id_source = Id::new(id_source);
+    pub fn id_salt(mut self, id_salt: impl Hash) -> Self {
```

Comment:
First of all: thanks for tackling this! But we need to keep the old functions also, with a `#[deprecated = "Renamed id_salt"]` attribute on them

Reply:
Okay! Did this - is this okay like that?